src/components/ListGroup/ListGroup.module.scss

```diff
@@ -1,15 +1,29 @@
 @import "~styles/variables.scss";
 
 .listItem {
-  padding: 10px 0;
-  border-bottom: 1px solid $base-border-color;
+  padding: $m-spacing $m-spacing;
+  position: relative;
+  border-radius: $base-border-radius;
+  margin-top: -1px;
```

Comment:
Same here - do we want to use spacing variables?

Reply:
Yep, I recently created a $base-border-width variable so will go through and replace instances like this 👍